.circleci/config.yml

```diff
@@ -126,7 +134,7 @@ jobs:
               export ACTION_VERSION=$(grep '"version":' package.json | cut -d\" -f4)
               git config --global user.name "Parabol CircleCI"
               git config --global user.email "admin+circleci@parabol.co"
-              git add queryMap.json build dist packages/client/__generated__ scripts/toolbox -f
+              git add queryMap.json build dist packages/client/__generated__ scripts/toolbox packages/server/postgres/queries/generated -f
```

Comment:
we won't be able to test this until it gets pushed to staging, but i'm pretty sure this should work